moped-editor/src/views/projects/projectView/ProjectComponents/CreateComponentDrawTools.js

```diff
@@ -42,7 +42,7 @@ const CreateComponentDrawTools = ({
   };
 
   const onUpdate = ({ features: updatedFeaturesArray, action }) => {
-    const wasComponentDragged = action === "move";
+    const wasComponentDragged = action === "move" || action === "change_coordinates";
```

Comment:
there are some additional special editing actions you can take: combine feature and uncombine/split features.

i did not attempt to enable these controls. i'm happy with only vertex editing for now ✨

Reply:
When I was testing this, I also remembered that the direct mode lets you create new vertices at the midpoint of lines. I think this fits what we want, but I wanted to make sure.


https://github.com/cityofaustin/atd-moped/assets/37249039/6687b059-b68f-4da2-a6d6-ff749fa9dea8

Reply:
oh—nice! will need to remember to show that to folks. or update the user docs 😵 
https://atd-dts.gitbook.io/moped-documentation/user-guides/project-map